doc/tutorial/ranges/index.md

```diff
@@ -208,21 +208,22 @@ Which of the following concepts do you think `v` models?
 | std::ranges::input_range         |   ✅    |
 | std::ranges::forward_range       |   ✅    |
 | std::ranges::bidirectional_range |   ✅    |
-| std::ranges::random_access_range |        |
-| std::ranges::contiguous_range    |        |
-|                                  |        |
+| std::ranges::random_access_range |         |
+| std::ranges::contiguous_range    |         |
+|                                  |         |
 | std::ranges::view                |   ✅    |
-| std::ranges::sized_range         |        |
-| std::ranges::output_range        |        |
+| std::ranges::sized_range         |         |
+| std::ranges::output_range        |         |
 
-The filter does not preserve RandomAccess and therefore not Contiguous, because it doesn't "know" which element
-of the underlying range is the i-th one in constant time.
-This also means we don't know the size.
+The filter does not preserve the RandomAccess concept because it doesn't "know" which element of the underlying range
+is the i-th one in constant time, therefore `v` does not model `std::ranges::random_access_range`. From this, it
+follows that `v` does not model `std::ranges::contiguous_range` nor `std::ranges::sized_range`, because both require
+RandomAccess.
 
 The transform on the other hand produces a new element on every access (the result of the multiplication), therefore
 `v` is not an output range, you cannot assign values to its elements.
-This would have prevented modelling the contiguous_range as well – if it hadn't been already by the filter – because
-values are created on-demand and are not stored in memory at all.
+Note that this prevents modelling the contiguous_range as well because values are created on-demand and are not stored
```

Comment:
```suggestion
Note that this prevents modelling the std::ranges::contiguous_range as well because values are created on-demand and are not stored
```